msdk-featdet/msdk-featdet-ADAP-3D/src/main/java/io/github/msdk/featdet/ADAP3D/ADAP3DFeatureDetectionMethod.java

```diff
@@ -108,11 +147,11 @@
    * builds Chromatogram for each good peak.
    * </p>
    * 
-   * @return featureList a list of {@link io.github.msdk.datamodel.impl.SimpleFeature}
+   * @return featureList a list of {@link io.github.msdk.datamodel.features.Feature}
    */
-  private List<SimpleFeature> getFeature(List<PeakDetection.GoodPeakInfo> goodPeakList) {
+  private List<Feature> getFeature(List<PeakDetection.GoodPeakInfo> goodPeakList,
```

Comment:
Could you give this function a more meaningful name then "getFeature"? 
Also, there is no need for this function to return the List<Feature>, if the list is provided as a parameter. Change the return type to void.